### PR TITLE
Expose attributes in MongoHttpSessionConfiguration

### PR DESCRIPTION
**Pull request by eddumelendez**
- [X] I have signed the CLA

Attributes collectionName and maxInactiveIntervalInSeconds can be set
now.

See gh-489

### CODE REVIEW

File: spring-session/src/integration-test/java/org/springframework/session/data/mongo/config/annotation/web/http/MongoHttpSessionConfigurationITests.java [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+import org.springframework.test.util.ReflectionTestUtils;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * @author Eddú Meléndez
+ * @since 1.2.0
+ */
+@ContextConfiguration
+public class MongoHttpSessionConfigurationITests extends AbstractMongoRepositoryITests {
```
Comment (vpavic) [inline spring-session/src/integration-test/java/org/springframework/session/data/mongo/config/annotation/web/http/MongoHttpSessionConfigurationITests.java]: This shouldn't be an integration test.

**Comment by eddumelendez** [comment]
@vpavic thanks for feedback.

@rwinch I have updated the PR.

**Comment by snicoll** [comment]
LGTM

File: spring-session/src/test/java/org/springframework/session/data/mongo/config/annotation/web/http/MongoHttpSessionConfigurationTest.java [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+import static org.mockito.Matchers.anyString;
+import static org.mockito.Mockito.mock;
+
+/**
+ * @author Eddú Meléndez
+ */
+public class MongoHttpSessionConfigurationTest {
+
+	private AnnotationConfigApplicationContext context = new AnnotationConfigApplicationContext();
+
```
Comment (rwinch) [inline spring-session/src/test/java/org/springframework/session/data/mongo/config/annotation/web/http/MongoHttpSessionConfigurationTest.java]: Should have an `@After` that closes the context

**Comment by eddumelendez** [comment]
@rwinch fixed

**Comment by rwinch** [comment]
Thanks for the PR! This is merged via edbf8bf58768747575f9fedffa379ec710fe1635

**Pull request closed without merging**